tests/functional/smarton/test_all.py

```diff
@@ -0,0 +1,33 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+
+from pages.smarton.base import SmartOnBasePage
+
+
+@pytest.mark.skip_if_firefox
+@pytest.mark.nondestructive
+@pytest.mark.parametrize('slug', [
+    pytest.mark.smoke(('tracking')),
+    ('security'),
+    ('surveillance')])
```

Comment:
Not tested, but if you put these two tests in a class, you may be able to have the `parameterize` mark just on the class and avoid some duplication.